migrations/emailmonitor/oxa_email_monitor.py

```diff
@@ -0,0 +1,215 @@
+    else:
+        os.system("echo '"+str(datetime.now())+" "+ log + ": "+str(error)+"' >> /oxa/oxa_email_monitor.log")
+		
+# This function creates the YYYY,MM,DD statistics raw in oxa.oxa_activationsummary table if it doesn't exists. This is rerunnable
+def create_year_month_day():
+    db = MySQLdb.connect(IP,User,Password,Database)
+    cursor = db.cursor()
+
+    sql = "SELECT YEAR(date_joined), MONTH(date_joined), DAY(date_joined), COUNT(*) FROM edxapp.auth_user \
+          GROUP BY DATE(date_joined) \
+          ORDER BY DATE(date_joined)"
+
+    cursor.execute(sql)
+    results = cursor.fetchall()
+    
+    for row in results:
+        y = str(row[0])
+        m = str(row[1])
+        d = str(row[2])
+        sql2 = "SELECT id FROM oxa.oxa_activationsummary WHERE xyear="+y+" and xmonth="+m+" and xday="+d
+        cursor.execute(sql2)
+        if cursor.rowcount < 1: 
+            sql3 = "INSERT INTO oxa.oxa_activationsummary (xyear, xmonth, xday) \
+                   VALUES ("+y+","+m+","+d+")"
+            cursor.execute(sql3) 
+
+    db.commit()
+    cursor.close()
+    db.close()
+
+# For each YYYY,MM,DD statistics raw in oxa.oxa_activationsummary table, this function updates the created accounts statistics based on edxapp.auth_user table. This is rerunnable
+def update_accounts_created():
+    db = MySQLdb.connect(IP,User,Password,Database)
+    cursor = db.cursor()
+
+    sql = "SELECT YEAR(date_joined), MONTH(date_joined), DAY(date_joined), COUNT(*) FROM edxapp.auth_user \
+          GROUP BY DATE(date_joined) \
+          ORDER BY DATE(date_joined)"
+
+    cursor.execute(sql)
+    results = cursor.fetchall()
+
+    for row in results:
+        y = str(row[0])
+        m = str(row[1])
+        d = str(row[2])
+        c = str(row[3])
+        sql2 = "UPDATE oxa.oxa_activationsummary SET newaccount="+c+" \
+               WHERE xyear="+y+" and xmonth="+m+" and xday="+d
+             
+        cursor.execute(sql2)
+
+    db.commit()
+    cursor.close()
+    db.close()
+
+# For each YYYY,MM,DD statistics raw in oxa.oxa_activationsummary table, this function updates the activated accounts statistics based on edxapp.auth_user table. This is rerunnable
+def update_accounts_activated():
+    db = MySQLdb.connect(IP,User,Password,Database)
+    cursor = db.cursor()
+
+    sql = "SELECT YEAR(date_joined), MONTH(date_joined), DAY(date_joined), COUNT(*) FROM edxapp.auth_user \
+          WHERE is_active=1 \
+          GROUP BY DATE(date_joined) \
+          ORDER BY DATE(date_joined)"
+
+    cursor.execute(sql)
+    results = cursor.fetchall()
+
+    for row in results:
+        y = str(row[0])
+        m = str(row[1])
+        d = str(row[2])
+        c = str(row[3])
+        sql2 = "UPDATE oxa.oxa_activationsummary SET activated="+c+" \
+               WHERE xyear="+y+" and xmonth="+m+" and xday="+d
+
+        cursor.execute(sql2)
+
+    db.commit()
```

Comment:
I did exception handling with SMTP but not with MySQL. If we get an error with MySQL the script will fail and will do nothing. I can add exception handling but don't know if it will worth.